mbuild/tests/test_coordinate_transform.py

```diff
@@ -67,23 +68,125 @@ def test_rigid_transform(self):
         rigid_transform = RigidTransform(A, B)
         assert (rigid_transform.apply_to(np.array([[2,3,4]])) == B).all()
 
-    @pytest.mark.skipif(True, reason="needs to be implemented")
+    def test_rotate_0(self, methane):
+        before = methane.xyz_with_ports
+        rotate(methane, 0.0, np.asarray([1.0, 0.0, 0.0]))
+        after = methane.xyz_with_ports
+        assert (np.array_equal(before, after))
+
+    def test_rotate_2pi(self, methane):
+        before = methane.xyz_with_ports
+        rotate(methane, 2*np.pi, np.asarray([1.0, 0.0, 0.0]))
+        after = methane.xyz_with_ports
+        assert (np.allclose(before, after))
+
+    def test_rotate_zero_vector(self, methane):
+        with pytest.raises(ValueError):
+            rotate(methane, np.pi/2, np.asarray([0.0, 0.0, 0.0]))
+            rotate(methane, 0.0, np.asarray([0.0, 0.0, 0.0]))
```

Comment:
You're right, they indeed don't get run. I'm removing them, since the extra runs aren't changing what I'm actually testing